ST_resgroups.c

```diff
@@ -203,16 +203,9 @@ static RealF _add_annuals(const GrpIndex rg, const SppIndex sp, const RealF last
     /*Get viable seeds from seed bank*/
     viable_seeds = (g->regen_ok) ? _get_annual_maxestab(sp) : 0;
 
-    /*Create a beta random number draw, where mean =s->seedling_estab_prob
-     variance = s->var. 
-     Calculate alpha and beta based on the mean and variance values by species.in*/
-     alpha = (pow (s->seedling_estab_prob, 2) - pow (s->seedling_estab_prob, 3)
-             - s->seedling_estab_prob * s->var) / s->var;
-     beta = (s->seedling_estab_prob - pow (s->seedling_estab_prob, 3) 
-             - s->var + s->var * s->seedling_estab_prob)/ s->var;
-     
-     /*Create beta random number*/
-     var = RandBeta(alpha, beta);
+    /*Create a beta random number draw based on alpha and beta for each species (calculated based on mean (s->seedling_estab_prob
+      * and variance (s->var)) */
+     var = RandBeta(Species[sp]->alpha, Species[sp]->alpha);
```

Comment:
I think the second argument should be `Species[sp]->beta`

Reply:
gawk! that was a copy and pasting error and is now fixed with below commit.

Reply:
https://github.com/DrylandEcology/STEPWAT2/pull/107/commits/43cf85273be12e5d89c51dfc1887d088e15d923e